src/components/features/NewFurniture/NewFurniture.js

```diff
@@ -19,15 +21,58 @@ class NewFurniture extends React.Component {
     this.setState({ activeCategory: newCategory });
   }
 
+  getDisplayedProductsCount(rwdMode) {
```

Comment:
Żeby zachować ciągłość nazewnictwa proponowałbym, żeby ostatni ELSE też był podpisany MODES.PHONE skoro go deklarujesz, tak żeby jak ktoś w to zajrzał wiedział od razu, że to tryb na mobilke ;) Może dla czystości kodu i ewentualnej prostoty wprowadzania w przyszłości innych trybów zamienić by warto if else na switch?